### ISSUE TITLE
Python 3.7 and pypy support

### ISSUE THREAD

**Issue by amenezes** [wikibusiness/aioamqp_consumer#19]
Hello guys.

I have tried use aioamqp_consumer with python 3.7 and there's a error that could be fixed simply updating aioamqp to version `0.11.0`.

```python
Python 3.7.6 (default, Jan 17 2020, 01:24:44) 
[GCC 9.2.1 20200104] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import aioamqp
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/amenezes/Documents/github/aioamqp_consumer/venv/lib/python3.7/site-packages/aioamqp/__init__.py", line 8, in <module>
    from .protocol import AmqpProtocol
  File "/home/amenezes/Documents/github/aioamqp_consumer/venv/lib/python3.7/site-packages/aioamqp/protocol.py", line 13, in <module>
    from .compat import ensure_future
  File "/home/amenezes/Documents/github/aioamqp_consumer/venv/lib/python3.7/site-packages/aioamqp/compat.py", line 11
    ensure_future = asyncio.async
                                ^
SyntaxError: invalid syntax

```

Also to verify and validate backwards compatibility I appended python 3.7 and pypy 3.6 in the tox.ini file as:

```ini
[tox]
envlist =
    py3{5,6,7},pypy3.6
...
```

All tests work's properly. But another issue maybe related to my environment is that tox didn't work at first only after an update (`pip install -U tox`).

```bash
py35: commands succeeded
py36: commands succeeded
py37: commands succeeded
pypy3.6: commands succeeded
congratulations :)
```

However, I noticed that project dosen't have updates more than a year ago. So this project still alive? A PR for this use case is relevant or there are any roadmap to include this support?

**Comment by hellysmile** [comment]
Thanks for the report, sorry for the late response, will be fixed by https://github.com/aio-libs/aioamqp_consumer/pull/20
